Editor/Editors/CinemachineSplineDollyEditor.cs

```diff
@@ -25,6 +69,35 @@ public override void OnInspectorGUI()
                 EditorGUILayout.HelpBox("AutoDolly requires a Follow Target", MessageType.Warning);
 
             DrawRemainingPropertiesInInspector();
+
+            EditorGUILayout.BeginHorizontal();
+            {
+                EditorGUILayout.PropertyField(m_CameraPosition, m_CameraPositionGUIContent);
+                EditorGUILayout.PropertyField(m_PositionUnits, m_PositionUnitsGUIContent, GUILayout.MaxWidth(100));
```

Comment:
why 100?  Can you find an "official" width for this?
Also, I'm not wild about the look of this gap:
![image](https://user-images.githubusercontent.com/6424658/154732366-9e7f0b1e-6f76-4c03-a504-e87bc634795a.png)

Reply:
I'll use UIToolkit then